src/bindings/interface/units.i

```diff
@@ -76,10 +80,10 @@ extract the units with the given `name`.";
 "Returns the scaling factor between two Units objects.";
 
 %feature("docstring") libcellml::Units::equivalent
-"Tests if two Units are equivalent.";
+"Tests if two Units are equivalent in base units as well as scaling factor.";
 
-%feature("docstring") libcellml::Units::dimensionallyEquivalent
-"Tests if two Units are dimensionally equivalent.";
+%feature("docstring") libcellml::Units::compatible
+"Tests if two Units have the same dimensional basis.";
```

Comment:
Minor: might want to swap `compatible` and `equivalent`, so we have the same order as in our C++ API.

Reply:
Done.